styles.js

```diff
@@ -11,6 +11,7 @@ export default createGlobalStyle`
   *::before,
   *::after {
     box-sizing: border-box;
+    /* outline: 1px solid black; */
```

Comment:
you can delete commented out code before committing :)

Reply:
Thank you for your feedback i will delete it now .